subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc

```diff
@@ -1,220 +1,196 @@
 `--single-path` (optional)::
-Indicates to render only a single path to the dependency.
-This might be useful to trim down the output in large graphs.
-
-[NOTE]
-====
-The `dependencyInsight` task selector will only execute the `dependencyInsight` task on a _single_ project.
-If you run the task on the root project, it will show the dependency information of the root project and not of any subproject.
-Be sure to always target the <<command_line_interface#executing_tasks_in_multi_project_builds, right project>> when running `dependencyInsight`.
-====
+Render only a single path to the dependency.
 
-=== Example: Using the dependency insight report for a given dependency
+The following code snippet demonstrates how to run a dependency insight report for a all paths to a dependency named "commons-codec" within the "scm" configuration:
 
-.Output of **`gradle -q dependencyInsight --dependency commons-codec --configuration scm`**
 ----
 > gradle -q dependencyInsight --dependency commons-codec --configuration scm
 include::{snippetsPath}/dependencyManagement/inspectingDependencies-dependencyInsightReport/tests/dependencyInsightReport.out[]
 ----
 
-As indicated above, omitting the `--configuration` parameter in a project that is not a Java project will lead to an error:
-----
-> Dependency insight report cannot be generated because the input configuration was not specified.
-  It can be specified from the command line, e.g: ':dependencyInsight --configuration someConf --dependency someDep'
-----
-
-For more information about configurations, see the documentation on declaring dependencies, which describes <<declaring_dependencies.adoc#sec:what-are-dependency-configurations, what dependency configurations are>>.
+For more information about configurations, see the <<declaring_dependencies.adoc#sec:what-are-dependency-configurations,dependency configuration documentation>>.
 
-=== Understanding selection reasons
+==== Selection Reasons
 
-The "Selection reasons" part of the dependency insight report will list the different reasons as to why a dependency was selected.
+The "Selection reasons" section of the dependency insight report lists the reasons why a dependency was selected.
 Have a look at the table below to understand the meaning of the different terms used:
 
-.Selections reasons terminology
+.Terminology
 [%header%autowidth,compact]
 |===
 | Reason    | Meaning
 
 | (Absent)
-| This means that no other reason than having a reference, direct or transitive, was present
+| No reason other than a reference, direct or transitive, was present.
 
 | Was requested : <text>
-| The dependency appears in the graph, and the inclusion came with a <<declaring_dependencies#sec:documenting-dependencies, `because` text>>.
+| The dependency appears and the inclusion came with a <<declaring_dependencies#sec:documenting-dependencies, `because` text>>.
```

Comment:
```suggestion
| The dependency appears in the graph, and the inclusion came with a <<declaring_dependencies#sec:documenting-dependencies, `because` text>>.
```